pyaedt/modules/PostProcessor.py

```diff
@@ -296,7 +296,7 @@ def rl_fields(self, expressions=None, setup_name=None, polyline=None):
         return
 
     @pyaedt_function_handler()
-    def far_field(self, expressions=None, setup_name=None, sphere_name=None):
+    def far_field(self, expressions=None, setup_name=None, sphere_name=None, source_context=None):
         """Create a Field Report object.
```

Comment:
Perhaps this should be "Create a far field object." The terminology here should be consistent with what is used later.